app/resources/results.html.template

```diff
@@ -46,16 +46,20 @@
       // third party search API directly here, to workaround AJAX's same
       // origin policy, and for authorization.
       function perform_third_party_search(id) {
-        jQuery.ajax({
-            url: repo_url + '/third_party_search',
-            data: {
-                query: query,
-                lang: lang,
-                query_type: query_type,
-                search_engine_id: id
-            },
-            dataType: 'json'
-        }).done(function(r) {
+        fetch(repo_url + '/third_party_search', {
+          method: 'POST',
```

Comment:
Just curious, why did you change this from GET to POST?

Reply:
I got some kind of error (can't remember what it was) trying to send the data with fetch as a GET request. Doesn't seem to be an issue with XMLHTTPRequest though (plus now that I think about it it doesn't really make sense for it to be a POST request), so changed it back to GET.